server/game/cards/13.4-BtRK/TowerOfTheHand.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class TowerOfTheHand extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => 
+                    event.card.controller === this.controller &&
+                    this.game.anyPlotHasTrait('Winter')
+            },
+            limit: ability.limit.perPhase(1),
+            target: {
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.isFaction('stark'),

```

Comment:
This will currently let you choose a character that is already standing. You'll need to add a `&& card.kneeled` check here.

(in the future it may not be required to explicitly check this, but for now it is)

Reply:
Thank you for your feedback. I added the check for the target card in the commit a2d776fd4ff4be69d8134c71819e4691c15bf9ad